nvchecker_source/pypi.py

```diff
@@ -19,6 +19,10 @@ async def get_version(name, conf, *, cache, **kwargs):
   data = await cache.get_json(url)
 
   for version in data['releases'].keys():
+    # Skip versions that are marked as yanked.
+    if len(data['releases'][version]) != 0 and data['releases'][version][0]['yanked']:
```

Comment:
```suggestion
    if (vers := data['releases'][version]) and vers[0]['yanked']:
```

Reply:
Done. I also added an explicit `python_requires` to the package metadata as the walrus operator was added in 3.8.